ZenPacks/zenoss/Import4/model.py

```diff
@@ -188,7 +197,12 @@ def _check_files(self):
 
         # check egg directories
         self.zenpack_count = int(subprocess.check_output(
```

Comment:
Here should be ok because there are two possible source of errors.
One possible error is the execution failure which will be captured by the exception. At that point, the system must gone very wrong.

If that's content error, the result will be "0", which will be handled later after it returns.